client/data/media/use-google-photos-picker-media-items-query.ts

```diff
@@ -0,0 +1,52 @@
+import { useQuery } from '@tanstack/react-query';
+import wp from 'calypso/lib/wp';
+
+interface Thumbnails {
+	large: string;
+	medium: string;
+	thumbnail: string;
+}
```

Comment:
Missing one `post-thumbnail` here.

Reply:
To double-check with you, `post_thumbnail` or `post-thumbnail`? Dash or underscore in between?